go.mod

```diff
@@ -5,6 +5,7 @@ go 1.13
 require (
 	github.com/BurntSushi/toml v0.3.1
 	github.com/Masterminds/semver v1.5.0
+	github.com/Masterminds/semver/v3 v3.1.1
```

Comment:
Do we need 2 different versions of this library?

Reply:
The v3 version branch allows you to use StrictNewVersion() this means that we won't match against some of the non semver tags in on the lifecycle image (ex. 9832537 in the original branch parses as 9832537.0.0 using the StrictNewVersion() function errors out on this tag). 

I could switch over to using a Regex but this felt more straight-forward

Reply:
I guess the question then is why do we need to use the older version elsewhere? I guess I'm just trying to see why we can't just use v3 everywhere.

Reply:
That's fair. I would just have to go around and convert everything and see if I can just swap everything over.

Reply:
@ryanmoran Alright, the latest commit should have upgraded everything to the v3 branch.